src/CsvColumnizer/CsvColumnizer.cs

```diff
@@ -47,7 +47,7 @@ public class CsvColumnizerConfig
 
         public void InitDefaults()
         {
-            delimiterChar = ';';

```

Comment:
I am not sure why this was semicolon instead of comma.

Reply:
Please let me know if I missed anything.

Reply:
Depends on the CSV file. There are some which are delimited by semicolon instead of comma. In my knowledge some people use the semicolon because its not used in "sentences" as for the comma its used often and needs some quotation.